far/FarEng.hlf.m4

```diff
@@ -5966,111 +5966,156 @@ operation is displayed, the faster the operation itself is performed.
+be used.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Default value: 0.
+
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @System.FlagPosixSemantics
 $ #far:config System.FlagPosixSemantics#
- Параметр "System.FlagPosixSemantics" задаёт поведение для процесса добавления отредактированного или просмотренного файла в историю.
-Если значение параметра равно true, то при поиске дублей учитывается регистр символов в именах файлов.
+ This Boolean parameter specifies whether inserting files into
+~view and edit history~@HistoryViews@ is case sensitive.
 
- По умолчанию значение = true.
+ If a file being added already exists in the history, it is not inserted
+again; instead, the existing history entry is moved to the most recent
+position.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ False - ^<wrap>Case insensitive comparison is used to search the
+history for duplicates.
+ True  - Case sensitive comparison is used to search the history for
+duplicates.
+
+ Default value: True (the search is case sensitive).
+
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @System.ShowCheckingFile
 $ #far:config System.ShowCheckingFile#
- Параметр "System.ShowCheckingFile" позволяет отображать в заголовке окна Far Manager имя плагина,
-претендующего на файл, который хотим запустить или отобразить в качестве файловой панели.
+ This Boolean parameter controls whether plugin’s name is displayed
+in the console window title while the plugin is checking a file.
 
- По умолчанию значение = false - не отображать информацию.
+ When the user presses #Enter# or #Ctrl+PgDn#, Far invokes registered
+plugins one by one to check if they can “open” or otherwise render the
+file. If this parameter is True, Far will show plugin’s name in the
+console window title while the plugin is checking the file.
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Default value: False (do not show plugins’ names).
+
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @System.PluginMaxReadData
 $ #far:config System.PluginMaxReadData#
- Параметр "System.PluginMaxReadData" позволяет задавать максимальный размер читаемых данных из
-файла в который попытались войти из панелей (Enter или Ctrl+PgDn). Считанные данные будут переданы
-плагинам для определения плагина поддерживающего файл этого типа.
+ This numeric parameter defines the maximum amount of file data used
+to find the plugin which supports the file.
 
- Минимальное значение - 0x1000. Максимальное - 0xFFFFFFFF.
+ When the user presses #Enter# or #Ctrl+PgDn#, Far reads the number
+of bytes specified by this parameter from the beginning of the file and
+passes the data to registered plugins to check if they can “open”
+or otherwise render the file.
 
- Не рекомендуется выставлять значение этого параметра больше 5 Mb.
+ Minimum value - 0x1000. Maximum value - 0xFFFFFFFF.
```

Comment:
Minimum value is currently 0x20000, maximum is limited only by the address space size (2<sup>32</sup> - 1 / 2<sup>64</sup> - 1).
Also, maybe use decimal?

Reply:
Thank you for pointing out. I should've checked myself. fixed.